docs/targets/index.md

```diff
@@ -10,20 +10,15 @@ The following is a list of Triggermesh event destinations known as `Targets`. So
     * [AWS SQS](./awssqs.md): Forward event payload to AWS Simple Queueing Service (SQS).
     * [AWS Kinesis](./awskinesis.md): Forward event payload to AWS Kinesis.
     * [AWS EventBridge](./awseventbridge.md): Forward arbitrary events to AWS EventBridge.
+* [Confluent](./confluent.md): Forward events to Confluent Kafka.
 * [Datadog](./datadog.md): Forward event payload to Datadog.
 * [Elasticsearch](./elasticsearch.md): Send events to Elasticsarch to be indexed.
+* [Googlesheets](./googlesheets.md): Send events to a GoogleSheets.
```

Comment:
Are we certain we want to show [this](https://docs.triggermesh.io/targets/googlesheets/) to the public?

Reply:
I am cleaning this up, and will be included in the next PR.